src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystemUpdater.java

```diff
@@ -58,6 +58,7 @@ private void updateControlledBus(VoltageControl voltageControl, EquationSystem<A
             // clean reactive power distribution equations
             controllerBuses.forEach(b -> equationSystem.removeEquation(b.getNum(), AcEquationType.ZERO_Q));
             // is there one static var compensator with a non-zero slope
+            // EquationTerm<AcVariableType, AcEquationType> vTerm = EquationTerm.createVariableTerm(controlledBus, AcVariableType.BUS_V, variableSet, controlledBus.getV().eval());
```

Comment:
This has to be checked. Why do I need this line in my first implementation of stand by automaton?